paddle/fluid/operators/listen_and_serv_op.cc

```diff
@@ -187,6 +187,7 @@ void ListenAndServOp::RunSyncLoop(framework::Executor *executor,
     for (auto &var : sparse_vars) {
       var->GetMutable<framework::SelectedRows>()->mutable_rows()->clear();
     }
+
```

Comment:
This change may not be nessesary.

Reply:
I am sorry about it, I will revert it later.